package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
> when you npm publish you will publish the entire dist/ folder so they will have the minified and non minified bundles.

Only if `dist/` is in files. That's what `files` is -- the files that get included in `npm publish`